frb_codegen/src/config.rs

```diff
@@ -78,9 +79,7 @@ pub struct Opts {
     pub manifest_path: String,
     pub dart_root: Option<String>,
     pub build_runner: bool,
-
-    // due to conflict of dart api of multiple blocks, keep it temporarily
-    pub exclude_sync_execution_mode_utility: bool,
+    pub block_index: usize, //NOTE: the index is counted from 1, not 0
```

Comment:
What about creating a `BlockIndex` newtype as I have suggested.

Reply:
> What about creating a `BlockIndex` newtype as I have suggested.

Fixed in [cc7cfbc](https://github.com/fzyzcjy/flutter_rust_bridge/pull/527/commits/cc7cfbc4f1a0b944bff5d25383e8f87edc1a914f).